src/core/public/apm_system.ts

```diff
@@ -29,6 +29,8 @@ interface StartDeps {
   application: InternalApplicationStart;
 }
 
+export const APP_CHANGE_TRANSACTION_TYPE = 'app-change';
```

Comment:
it's not used outside of the file. please, inline into the call
```js
 this.apm.startTransaction(appId, 'app-change', {
```

Reply:
Well, it should be used in tests :-}

Reply:
tests can inline it as well

Reply:
👍🏻